src/components/image-url-input.js

```diff
@@ -0,0 +1,75 @@
+import React, { useMemo } from 'react';
+import PropTypes from 'prop-types';
+import ReactCSSTransitionGroup from 'react-addons-css-transition-group';
+import styled from 'styled-components/macro';
+import { Flex } from 'rebass';
+
+import { Input } from './';
+
+const ImageUrlInput = ({ visible, value, onChange, transitionName }) => {
```

Comment:
You can move `value` and `onChange` to `...rest` and then spread them on the actual input element.